### PR TITLE
feat(projection-service): collection ground to image

### PR DESCRIPTION
**Pull request by yairtawil**
`projectCollectionAccurately` and `projectCollectionApproximately` to projection service

### CODE REVIEW

File: src/app/@ansyn/plugins/openlayers/open-layer-visualizers/tools/measure-distance.visualizer.ts [hunk 1]
```diff
@@ -135,16 +136,16 @@ export class MeasureDistanceVisualizer extends EntitiesVisualizer {
...
-		let featureJson = <any> this.geoJsonFormat.writeFeatureObject(data.feature, {
-			featureProjection,
-			dataProjection: 'EPSG:4326',
-		});
-		const newEntity: IVisualizerEntity = {
-			id: UUID.UUID(),
-			featureJson
-		};
-		this.addOrUpdateEntities([newEntity]);
+		// const featureProjection = view.getProjection();
```
Comment (liorze) [inline src/app/@ansyn/plugins/openlayers/open-layer-visualizers/tools/measure-distance.visualizer.ts]: nit: remove this comment

File: src/app/@ansyn/plugins/openlayers/open-layers-map/openlayers-map/openlayers-map.ts [hunk 2]
```diff
@@ -98,13 +98,13 @@ export class OpenLayersMap extends IMap<OLMap> {
 		this.initMap(element, _mapLayers, position);
 	}
 
-	public positionToPoint(coordinate: ol.Coordinate, cb: (p: GeoJSON.Point) => void) {
+	public positionToPoint(coordinates: ol.Coordinate, cb: (p: GeoJSON.Point) => void) {
```
Comment (liorze) [inline src/app/@ansyn/plugins/openlayers/open-layers-map/openlayers-map/openlayers-map.ts]: It's a single coordinate, I think the naming here is ok.

File: src/app/@ansyn/plugins/openlayers/open-layers-map/openlayers-map/openlayers-map.ts [hunk 3]
```diff
@@ -425,7 +425,7 @@ export class OpenLayersMap extends IMap<OLMap> {
 	// *****-- pointer move --********
 
 	public onPointerMove(e) {
-		this.positionToPoint(e.coordinate, p => this.pointerMove.emit({lonLat: p.coordinates}));
+		this.positionToPoint(e.coordinate, p => this.pointerMove.emit(p.coordinates));
```
Comment (liorze) [inline src/app/@ansyn/plugins/openlayers/open-layers-map/openlayers-map/openlayers-map.ts]: :+1:

File: src/app/@ansyn/imagery/projection-service/projection.service.ts [hunk 4]
```diff
@@ -1,15 +1,25 @@
 import { Observable } from 'rxjs/Observable';
-import { GeometryObject, Point, Position } from 'geojson';
+import { FeatureCollection, GeometryObject, Point, Position } from 'geojson';
 import { IMap } from '@ansyn/imagery/index';
 
 export abstract class ProjectionService {
```
Comment (liorze) [inline src/app/@ansyn/imagery/projection-service/projection.service.ts]: We should probably revisit this API in the future. It's not designed well at this moment.

File: src/app/@ansyn/plugins/openlayers/open-layers-map/openlayers-map/openlayers-map.ts (hunk 2 above)
Comment (yairtawil) [reply src/app/@ansyn/plugins/openlayers/open-layers-map/openlayers-map/openlayers-map.ts]: GeoJSON.Point = {type, coordinates} never `coordinate`

File: src/app/@ansyn/imagery/projection-service/projection.service.ts (hunk 4 above)
Comment (yairtawil) [reply src/app/@ansyn/imagery/projection-service/projection.service.ts]: 👍

**Pull request merged**